install/kubernetes/helm/istio/values.yaml

```diff
@@ -8,19 +8,22 @@ global:
   # Default repository for Istio images.
   # Releases are published to docker hub under 'istio' project.
   # Daily builds from prow are on gcr.io, and nightly builds from circle on
-  # docker.io/istionightly
-  hub: docker.io/istio
+  hub: docker.io/istionightly
+  #hub: docker.io/istio
```

Comment:
is this leftover?

Reply:
Same.. I can remove the commented out - and have each branch have it's own version.